### ISSUE TITLE
Set up use of Postgres without using any ORM

### ISSUE THREAD

**Issue by ezkemboi** [kemboijs/nodejs-api-cli#17]
When using a database, there are some people that do not need to use ORM.
Thus, we need settings for the same in the best way possible. That is, set up Postgres with no use of ORM.

**Comment by hezronkimutai** [comment]
Hi, In the past two days, I have gone through the codebase. I have some concerns. 
- I have used `noOrm ` to indicate that the user does not require an ORM. However, I am still looking for a better word to use
```

        const { database } = collectFrameworkAndDb;

        let ormChoices = ['No ORM'];

``` 
- I have seen that `src/models` file is created. Should I use this folder to write in the setup for database connection without ORM or I should add another folder `src/database`
- I am also curious how I should test the application without necessarily publishing it.

**Comment by ezkemboi** [comment]
I hope we clarified the above issues/items in the call we had.

**Comment by hezronkimutai** [comment]
Yeah. All the issues I raised were very clear. Thanks.

**Comment by ezkemboi** [comment]
We can close this issue when we are done merging the PR that is open @hezronkimutai

**Comment by ezkemboi** [comment]
closed by #46
